app/src/main/java/org/simple/clinic/sync/RemoteConfigSyncWorker.kt

```diff
@@ -0,0 +1,50 @@
+package org.simple.clinic.sync
+
+import android.content.Context
+import androidx.work.Constraints
+import androidx.work.NetworkType
+import androidx.work.OneTimeWorkRequest
+import androidx.work.Worker
+import androidx.work.WorkerParameters
+import org.simple.clinic.ClinicApp
+import org.simple.clinic.remoteconfig.RemoteConfigService
+import javax.inject.Inject
+
+class RemoteConfigSyncWorker(
```

Comment:
Since we already use the "Sync" to refer to the domain specific sync resources, can we use a different name for this so that it is clear the two are not related? Something like `UpdateRemoteConfigWorker` or something similar?

Similar thoughts on the package naming as well.